packages/gatsby-plugin-image/src/resolver-utils.ts

```diff
@@ -77,34 +77,34 @@ export function getGatsbyImageFieldConfig<TSource, TContext>(
         type: ImagePlaceholderType,
         defaultValue: `blurred`,
         description: stripIndent`
-            Format of generated placeholder image, displayed while the main image loads. 
+            Format of generated placeholder image, displayed while the main image loads.
             BLURRED: a blurred, low resolution image, encoded as a base64 data URI (default)
-            DOMINANT_COLOR: a solid color, calculated from the dominant color of the image. 
+            DOMINANT_COLOR: a solid color, calculated from the dominant color of the image.
             TRACED_SVG: a low-resolution traced SVG of the image.
             NONE: no placeholder. Set "background" to use a fixed background color.`,
       },
       formats: {
         type: GraphQLList(ImageFormatType),
         description: stripIndent`
-            The image formats to generate. Valid values are AUTO (meaning the same format as the source image), JPG, PNG, WEBP and AVIF. 
+            The image formats to generate. Valid values are AUTO (meaning the same format as the source image), JPG, PNG, WEBP and AVIF.
             The default value is [AUTO, WEBP], and you should rarely need to change this. Take care if you specify JPG or PNG when you do
-            not know the formats of the source images, as this could lead to unwanted results such as converting JPEGs to PNGs. Specifying 
+            not know the formats of the source images, as this could lead to unwanted results such as converting JPEGs to PNGs. Specifying
             both PNG and JPG is not supported and will be ignored. AVIF support is currently experimental.
         `,
-        defaultValue: [`auto`, `webp`],
+        defaultValue: [``, `webp`],
```

Comment:
is this change intentional?

Reply:
Yes, there is no value `auto` in this enum (there is a key `AUTO` but its value is `""`). And `graphql-js` now throws when it sees non-existent enum value (after graphql/graphql-js#1827)

Reply:
I think it was working before by a pure accident 😅

Reply:
Huh, ok ;) just thinking about readability - would it make sense to do this other way around - set `AUTO` key to `auto` value instead of empty string? :P

Reply:
I was thinking about it but this has bigger potential to introduce some bug. See the way this type is defined is pretty weird:

https://github.com/gatsbyjs/gatsby/blob/b93b8bd90fb450518d974f199a83921b97a3715d/packages/gatsby-transformer-sharp/src/types.ts#L17-L22

`NO_CHANGE` and `AUTO` have the same value (empty string). If you only change `AUTO` - then it will change the behavior in one way or another. I didn't want to mess too much with image plugins in this PR so this was a safer choice.

Reply:
This was for backward-compatibility reasons.